### PR TITLE
[Docs] Add zh_cn document of ONNX

### PR DESCRIPTION
**Pull request by grimoire**
## Motivation

This PR add zh_cn document of ONNX and ONNX Runtime custom ops.

## Modification

`docs_zh_CN/deployment/onnx.md`, 
`docs_zh_CN/deployment/onnxruntime_custom_ops.md`,
`docs_zh_CN/deployment/onnxruntime_op.md`
has been added.

### CODE REVIEW

File: docs_zh_CN/deployment/onnxruntime_custom_ops.md [hunk 1]
```diff
@@ -1,3 +1,333 @@
...
+
+### 描述
+
+根据`scores`计算`boxes`的soft NMS。 请阅读[Soft-NMS -- Improving Object Detection With One Line of Code](https://arxiv.org/abs/1704.04503)了解细节。
+
+### 模型参数
+
+| 类型    | 参数名          | 描述                                                    |
+| ------- | --------------- | ------------------------------------------------------- |
+| `float` | `iou_threshold` | 用来判断候选框覆盖度的阈值，取值范围[0, 1]。默认值为0   |
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/onnxruntime_custom_ops.md:67]:
```suggestion
| `float` | `iou_threshold` | 用来判断候选框重合度的阈值，取值范围[0, 1]。默认值为0   |
```

File: docs_zh_CN/deployment/onnxruntime_custom_ops.md [hunk 2]
```diff
@@ -1,3 +1,333 @@
...
+<dt><tt>boxes</tt>: T</dt>
+<dd>输入候选框。形状为(N, 4)的二维张量，N为候选框数量。</dd>
+<dt><tt>scores</tt>: T</dt>
+<dd>输入得分。形状为(N, )的一维张量。</dd>
+</dl>
+
+### 输出
+
+<dl>
+<dt><tt>dets</tt>: T</dt>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/onnxruntime_custom_ops.md:85]:
```suggestion
<dt><tt>dets</tt>: tensor(int64)</dt>
```

File: docs_zh_CN/deployment/onnxruntime_custom_ops.md [hunk 3]
```diff
@@ -1,3 +1,333 @@
...
+
+### 类型约束
+
+- T:tensor(float32)
+
+## RoIAlign
+
+### 描述
+
+在特征图上计算RoIAlign，在多数双阶段目标检测模型的bbox_head中使用。
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/onnxruntime_custom_ops.md:99]:
```suggestion
在特征图上计算RoIAlign，通常在双阶段目标检测模型的bbox_head中使用
```

File: docs_zh_CN/deployment/onnx.md [hunk 4]
```diff
@@ -1,3 +1,19 @@
-# MMCV 中的 onnx 模块 (实验性质)
+# MMCV中ONNX模块简介 (实验性)
 
-欢迎有兴趣的朋友一起翻译 MMCV 文档。如有兴趣，请在 [MMCV issue](https://github.com/open-mmlab/mmcv/issues) 提 issue 确定翻译的文档。
+## register_extra_symbolics
+
+在将PyTorch模型导出成ONNX时，需要注册额外的符号函数。
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/onnx.md:5]:
```suggestion
在将PyTorch模型导出成ONNX时，需要注册额外的符号函数
```

File: docs_zh_CN/deployment/onnxruntime_custom_ops.md [hunk 5]
```diff
@@ -1,3 +1,333 @@
...
+
+### 模型参数
+
+| 类型    | 参数名           | 描述                                                    |
+| ------- | ---------------- | ------------------------------------------------------- |
+| `int`   | `output_height`  | 输出roi特征的高度                                       |
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/onnxruntime_custom_ops.md:105]:
```suggestion
| `int`   | `output_height`  | roi特征的输出高度                                       |
```

File: docs_zh_CN/deployment/onnxruntime_custom_ops.md [hunk 6]
```diff
@@ -1,3 +1,333 @@
...
+| `int`   | `output_width`   | 输出roi特征的宽度                                       |
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/onnxruntime_custom_ops.md:106]:
```suggestion
| `int`   | `output_width`   | roi特征的输出宽度                                       |
```

File: docs_zh_CN/deployment/onnxruntime_custom_ops.md [hunk 7]
```diff
@@ -1,3 +1,333 @@
...
+
+### 描述
+
+根据IoU阈值对候选框进行非极大值抑制。
+
+### 模型参数
+
+| 类型    | 参数名          | 描述                                                  |
+| ------- | --------------- | ----------------------------------------------------- |
+| `float` | `iou_threshold` | 用来判断候选框覆盖度的阈值，取值范围[0, 1]。默认值为0 |
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/onnxruntime_custom_ops.md:142]:
```suggestion
| `float` | `iou_threshold` | 用来判断候选框重合度的阈值，取值范围[0, 1]。默认值为0 |
```

File: docs_zh_CN/deployment/onnxruntime_custom_ops.md [hunk 8]
```diff
@@ -1,3 +1,333 @@
...
+<dd>输入候选框。形状为(N, 4)的二维张量，N为候选框数量。</dd>
+<dt><tt>scores</tt>: T</dt>
+<dd>输入得分。形状为(N, )的一维张量。</dd>
+</dl>
+
+### 输出
+
+<dl>
+<dt><tt>indices</tt>: tensor(int32, Linear)</dt>
+<dd>被选中的候选框序号。形状为(num_valid_boxes, )的一维张量，num_valid_boxes表示被选上的候选框数量。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/onnxruntime_custom_ops.md:158]:
```suggestion
<dd>被选中的候选框索引。形状为(num_valid_boxes, )的一维张量，num_valid_boxes表示被选上的候选框数量。</dd>
```

File: docs_zh_CN/deployment/onnxruntime_custom_ops.md [hunk 9]
```diff
@@ -1,3 +1,333 @@
...
+
+### 类型约束
+
+- T:tensor(float32)
+
+## grid_sampler
+
+### 描述
+
+根据像素位置网格`grid`对`input`进行网格采样。
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/onnxruntime_custom_ops.md:169]:
```suggestion
根据`grid`的像素位置对`input`进行网格采样。
```

File: docs_zh_CN/deployment/onnxruntime_custom_ops.md [hunk 10]
```diff
@@ -1,3 +1,333 @@
...
+
+### 模型参数
+
+| 类型  | 参数名               | 描述                                                                                                                                                   |
+| ----- | -------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `int` | `interpolation_mode` | 计算输出使用的插值模式。(0: `bilinear` , 1: `nearest`)                                                                                                 |
+| `int` | `padding_mode`       | 边缘填充模式。(0: `zeros`, 1: `border`, 2: `reflection`)                                                                                               |
+| `int` | `align_corners`      | 如果`align_corners=1`，则极值(`-1`和`1`)会被当做输入边缘像素的中心点。如果`align_corners=0`，则他们会被看做是边缘像素的边缘点,减小分辨率对采样的影响。 |
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/onnxruntime_custom_ops.md:177]:
```suggestion
| `int` | `align_corners`      | 如果`align_corners=1`，则极值(`-1`和`1`)会被当做输入边缘像素的中心点。如果`align_corners=0`，则它们会被看做是边缘像素的边缘点,减小分辨率对采样的影响。 |
```

File: docs_zh_CN/deployment/onnxruntime_custom_ops.md [hunk 11]
```diff
@@ -1,3 +1,333 @@
...
+
+| 类型  | 参数名 | 描述               |
+| ----- | ------ | ------------------ |
+| `int` | `dim`  | 进行累计计算的维度 |
+
+### 输入
+
+<dl>
+<dt><tt>input</tt>: T</dt>
+<dd>输入张量；可以使任意形状</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/onnxruntime_custom_ops.md:277]:
```suggestion
<dd>输入张量；可以是任意形状；也支持空 tensor</dd>
```

File: docs_zh_CN/deployment/onnxruntime_custom_ops.md [hunk 12]
```diff
@@ -1,3 +1,333 @@
...
+</dl>
+
+### 输出
+
+<dl>
+<dt><tt>output</tt>: T</dt>
+<dd>`input`第`dim`维的累计最小值，形状与`input`相同。</dd>
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/onnxruntime_custom_ops.md:284]:
```suggestion
<dd>`input`第`dim`维的累计最小值，形状与`input`相同。类型和`input`一致</dd>
```

File: docs_zh_CN/deployment/onnxruntime_op.md [hunk 13]
```diff
@@ -1,3 +1,127 @@
...
+
+- 该算子的ONNX Runtime实现尚未在MMCV中支持[已实现算子列表](https://github.com/microsoft/onnxruntime/blob/master/docs/OperatorKernels.md)。
+- 确保该自定义算子可以被ONNX导出。
+
+### 添加方法
+
+以`soft_nms`为例：
+
+1. 在ONNX Runtime头文件目录`mmcv/ops/csrc/onnxruntime/`下添加头文件`soft_nms.h`
+2. 在ONNX Runtime实现目录`mmcv/ops/csrc/onnxruntime/cpu/`下添加算子实现`soft_nms.cpp`
```
Comment (zhouzaida) [inline docs_zh_CN/deployment/onnxruntime_op.md:100]:
```suggestion
2. 在ONNX Runtime源码目录`mmcv/ops/csrc/onnxruntime/cpu/`下添加算子实现`soft_nms.cpp`
```

File: docs_zh_CN/deployment/onnxruntime_custom_ops.md (hunk 2 above)
Comment (grimoire) [reply docs_zh_CN/deployment/onnxruntime_custom_ops.md:85]: Output type should be float32 since dets[:,4] is score.

Comment (zhouzaida) [reply docs_zh_CN/deployment/onnxruntime_custom_ops.md:85]: Got it. Maybe you can modify the English version(https://github.com/open-mmlab/mmcv/edit/master/docs/deployment/onnxruntime_custom_ops.md) which is `tensor(int64)`

**Review by zhouzaida — APPROVED** [review]
LGTM

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**